cpp/ql/src/Security/CWE/CWE-190/ArithmeticUncontrolled.ql

```diff
@@ -5,7 +5,7 @@
  * @kind path-problem
  * @problem.severity warning
  * @security-severity 8.6
- * @precision high
+ * @precision medium
```

Comment:
Reducing the precision of a query from `high` to `medium` means it's no longer running as part of Code Scanning. In the past we've not done such drastic changes, so I don't think we should do that as part of this PR.

If you're concerned with the new FP on the query then we could put a barrier on all dataflow nodes with a floating point types in the configuration [here](https://github.com/github/codeql/blob/40b75342104ec6a449750332af0d81a501cf9ce0/cpp/ql/src/Security/CWE/CWE-190/ArithmeticUncontrolled.ql#L108).

Reply:
Ok, you can check out the latest commit.

Reply:
Awesome! That LGTM 🎉